spec/models/vacancy_spec.rb

```diff
@@ -562,6 +562,34 @@
     end
   end
 
+  describe "#distance_in_miles_to" do
+    let(:test_coordinates) { Geocoding.new("Stonehenge").coordinates }
+    subject { create(:vacancy, organisations: organisations) }
+
+    context "when vacancy has multiple geolocations" do
+      let(:glasgow_school) { create(:school, geopoint: "POINT(-4.2542 55.8628)") }
+      let(:manchester_school) { create(:school, geopoint: "POINT(-2.2374 53.4810)") }
+      let(:canary_wharf_school) { create(:school, geopoint: "POINT(-0.019501 51.504949)") }
+      let(:organisations) { [glasgow_school, manchester_school, canary_wharf_school] }
+      let(:miles_from_glasgow_to_stonehenge) { 338 }
```

Comment:
We don't need these values for the test to run but I thought it was helpful to explicitly show which school was the closest.